var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -650,6 +656,17 @@ def define_tpl_enable(cmake_var, spec_var=None):
                 define('SuperLU_INCLUDE_DIRS', spec['superlu'].prefix.include),
             ])
 
+        options.append(define_tpl_enable('STRUMPACK'))
+        if '+strumpack' in spec:
+            options.extend([
+                define('TPL_ENABLE_STRUMPACK', True),
+                define('Amesos2_ENABLE_STRUMPACK', True),
+                define('STRUMPACK_LIBRARY_DIRS',
+                       spec['strumpack'].prefix.lib),
```

Comment:
```suggestion
                       spec['strumpack'].libs.directories[0]),
```
Libraries may be in `lib` or `lib64`